src/components/Card.tsx

```diff
@@ -0,0 +1,28 @@
+"use client";
+import { Album, Song } from "@/db/schema";
+import { DeleteAlbum, DeleteSong } from "./Delete";
+import Link from "next/link";
+
+export function AlbumCard({ album }: { album: Album }) {
+  return (
+    <div className="flex flex-col rounded border m-3 hover:bg-blue-200 p-3 w-1/2">
+      <DeleteAlbum id={album.id} />
+      <Link className="font-bold text-xl" href={`./album/${album.id}`}>
```

Comment:
never ever use absolute paths, i already told it to you at some point. This will behave differently if the component is rendered on different routes, which can lead to super weird bugs...
```suggestion
      <Link className="font-bold text-xl" href={`/album/${album.id}`}>
```